testcommon/testInitializer_vmoutput.go

```diff
@@ -230,27 +246,30 @@ func (v *VMOutputVerifier) Transfers(transfers ...TransferEntry) *VMOutputVerifi
 
 	for _, outputAccount := range v.VmOutput.OutputAccounts {
 		transfersForAccount := transfersMap[string(outputAccount.Address)]
-		require.Equal(v.T, len(transfersForAccount), len(outputAccount.OutputTransfers), "Transfers")
+		require.Equal(v.T, len(transfersForAccount), len(outputAccount.OutputTransfers), getErrorForAccount("Transfers to ", outputAccount.Address))
 		for idx := range transfersForAccount {
-			require.Equal(v.T, transfersForAccount[idx], outputAccount.OutputTransfers[idx], "Transfers")
+			require.Equal(v.T, transfersForAccount[idx], outputAccount.OutputTransfers[idx], getErrorForAccount("Transfers from / to ", outputAccount.OutputTransfers[idx].SenderAddress, outputAccount.Address))
```

Comment:
not a big fan of long code lines
some formatting can make this a bit easier to read.